internal/bucket_test.go

```diff
@@ -94,9 +94,9 @@ func TestBucketsCreate(t *testing.T) {
 		{
 			name: "retention but not shard-group duration",
 			params: internal.BucketsCreateParams{
```

Comment:
Can you add tests for:
* Create w/ `explicit`
* List w/ combo of `implicit` and `explicit`?

Instead of new tests, I think it'd be ok to modify some of the existing ones.

Reply:
Done